_posts/2015-03-23-CocoaDocs-Colo.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title:  "CocoaDocs Co-located"
+author: orta
+categories: cocoadocs.org releases website
+---
+
+TL;DR: [CocoaDocs.org][1] just got [Jazzy][4] support, became more stable, and is now sending stats to our status page.
+
+<!-- more -->
+
+## History.
+
+CocoaDocs is a server that generates CocoaPods documentation. It started out by generating Apple-compliant DocSets via [appledoc][2] on a hosted Mac and then sends the results to Amazon's S3 where they are served statically.
+
+## Maturation Process
+
+Appledoc is great, but it's not going to be supporting Swift anytime soon. It's a pre-ARC project that isn't compilable with modern Xcodes. CocoaPods has been supporting Swift Pods since New Year 2014, we wanted to be able to provide hosted documentation for these pods too. Sam Giddins took the initiative here and added [early support][3] for Swift Pods via [Realm][8]'s [Jazzy][4]. You can see a few Pods here: [Observable](http://cocoadocs.org/docsets/Observable-Swift/0.4.2/) , [Moya](http://cocoadocs.org/docsets/Moya/0.6.1/), [JSONJoy](http://cocoadocs.org/docsets/JSONJoy-Swift/0.9.1/) and [QRReader.swift](http://cocoadocs.org/docsets/QRCodeReader.swift/3.1.7/). There's no fancy design work yet, we will get to that in time.
```

Comment:
> It's a pre-ARC project that isn't compilable with modern Xcodes

That was true for a long time, but fixed as of a few months ago. Appledoc builds fine with Xcode 5+ now.